packages/strapi-connector-bookshelf/lib/knex.js

```diff
@@ -97,7 +97,9 @@ module.exports = strapi => {
             socketPath: _.get(connection.settings, 'socketPath'),
             ssl: _.get(connection.settings, 'ssl', false),
             timezone: _.get(connection.settings, 'timezone', 'utc'),
-            filename: _.get(connection.settings, 'filename', '.tmp/data.db'),
+            filename: _.get(connection.settings, 'filename')
```

Comment:
Sorry but we don't want to ingore user errors as it can lead to unexpected behavior down the line.

Reply:
"How did you endup having the set to an empty string ?" 
When installing strapi  i don't selected data.db(click up arrow on linux)

Reply:
Yeah that's What I figured. So It would make more sense to set the default in the cli package if the user doesn't define one instead :)

Reply:
but _.get() not working in your code with '' or null. Isn't that a mistake?

Reply:
This is intended. We don't want to overwritte a config set in the user files. It is better to have it fail early so the user fix their config issues instead of trying to copensate in this location & have a bug somewhere later in the code. 
We could improve the error handling to make it better for the user to fix their config though !
Hope you see the point.

Reply:
okay, I got it